.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ version: 2
 jobs:
   unit_test:
     docker:
-      - image: circleci/ruby:2.7.4-buster-node-browsers
+      - image: cimg/ruby:2.7.4-node
```

Comment:
Where are you accessing the images? I feel I have missed the boat on this. I was still looking in dockerhub for everything.

Reply:
CircleCi has their own "Cimg" (and their legacy circleci/* images) available. Google "cimg <lang>" to get a list of each!